src/host/screenInfo.cpp

```diff
@@ -1908,7 +1884,7 @@ const SCREEN_INFORMATION& SCREEN_INFORMATION::GetMainBuffer() const
         auto& myCursor = GetTextBuffer().GetCursor();
         auto* const createdBuffer = *ppsiNewScreenBuffer;
         auto& altCursor = createdBuffer->GetTextBuffer().GetCursor();
-        altCursor.SetStyle(myCursor.GetSize(), myCursor.GetColor(), myCursor.GetType());
+        altCursor.SetStyle(myCursor.GetSize(), myCursor.GetType());

```

Comment:
Now that the cursor color is stored in the color table, it doesn't need to be copied backwards and forwards between the alt buffer cursor and the main buffer - it's always the same everywhere.